redis/src/aio.rs

```diff
@@ -64,6 +64,7 @@ pub(crate) trait RedisRuntime: AsyncStream + Send + Sync + Sized + 'static {
         hostname: &str,
         socket_addr: SocketAddr,
         insecure: bool,
+        ca_cert: &Option<String>,
```

Comment:
This should take an `Option<&[u8]>` (`&` on the inside) instead.

Reply:
upated